tests/lib/rules/sort-keys.js

```diff
@@ -33,9 +33,18 @@ ruleTester.run("sort-keys", rule, {
         // ignore non-simple computed properties.
         { code: "var obj = {a:1, b:3, [a + b]: -1, c:2}", options: [], parserOptions: { ecmaVersion: 6 } },
 
-        // ignore spread properties.
+        // ignore non-boundary spread properties.
```

Comment:
I have the same suggestion about the term "non-boundary" here -- maybe something like "properties separated by spread properties" would be better.